audiofile/core/io.py

```diff
@@ -129,26 +128,26 @@ def write(
     file_type = file_extension(file)
     # Check for allowed precisions
     if file_type == 'wav':
-        precision_mapping = {
-            '8bit': 'PCM_U8',
-            '16bit': 'PCM_16',
-            '24bit': 'PCM_24',
-            '32bit': 'PCM_32',
+        depth_mapping = {
+            8: 'PCM_U8',
```

Comment:
Before it was `PCM_S8`. Why?

Reply:
oh wait, sorry. my bad, it's `PCM_S8` for FLAC.

Reply:
It's different for FLAC and WAV.